pubspec.yaml

```diff
@@ -7,13 +7,14 @@ version: 1.0.0+1
 environment:
   sdk: '>=3.2.3 <4.0.0'
 
-dependencies:
-  flutter:
-    sdk: flutter
-
-  provider: ^6.1.2
-  csv: ^6.0.0
-  http: ^1.5.0
+dependencies:
+  flutter:
+    sdk: flutter
+
+  cupertino_icons: ^1.0.8
```

Comment:
Umm, wenn wir bei material bleiben könnten wär mir das ganz lieb. Bisher ist die ganze app in material gemacht, und ich finde, wir sollten da konsistent bleiben.